### PR TITLE
Task/add sim controller

### PR DESCRIPTION
**Pull request by lassombra**
This adds a sim controller if the contents of the prefab require one.

It also adds InteractablePortFeedersController and IndicatorPortReadersController as needed

### CODE REVIEW

File: CCL.Importer/PrefabWrangler.cs [hunk 1]
```diff
@@ -65,10 +69,66 @@ private static bool FinalizeLiveryPrefab(CCL_CarVariant livery)
...
             CCLPlugin.Log($"Finalized prefab for {livery.id}");
             return true;
         }
 
+        //==============================================================================================================
+        #region SimulationComponents - SimController and friends
+        private static void BuildSimulationElements(TrainCarLivery livery)
+        {
+            AddAdditionalControllers(livery.prefab);
+            if (livery.interiorPrefab != null)
```
Comment (katycat5e) [inline CCL.Importer/PrefabWrangler.cs:84]: For anything inheriting from UnityEngine.Object we can use the implicit bool conversion - `if (livery.interiorPrefab)`, I think it's recommended over a check against null

**Review by katycat5e — APPROVED** [review]
Looks good. My only other recommendation would be to add some whitespace between blocks within methods but that's just aesthetic

File: CCL.Importer/PrefabWrangler.cs (hunk 1 above)
Comment (lassombra) [reply CCL.Importer/PrefabWrangler.cs:84]: I was unaware of that capability - I'll make those changes later today

**Comment by katycat5e** [comment]
Superceded by #169

**Pull request closed without merging**